Bento/Views/BentoReusableView.swift

```diff
@@ -1,10 +1,20 @@
+/// Existential type for reusable views (cell, header, footer) in `UITableView`, `UICollectionView`, etc.
 protocol BentoReusableView: AnyObject {
-    var containedView: UIView? { get set }
+    /// The main view to which you add your reusable view’s custom content.
+    /// - Note: This is normally system-provided as `self`'s subview.
```

Comment:
This should be a precondition.

Reply:
I think we don't need to force this as precondition since this could be useful in other view-reusing library which might not have the same variable name.

Reply:
`contentView` is not always `self` e.g `UITableViewCell` and `UITableViewHeaderFooterView`

Reply:
@sergdort  Here I wrote `contentView` as `self`'s subview, not `self`.